variables.tf

```
@@ -119,3 +119,39 @@ variable "regex_replace_chars" {
   default     = "/[^a-zA-Z0-9-]/"
   description = "Regex to replace chars with empty string in `namespace`, `environment`, `stage` and `name`. By default only hyphens, letters and digits are allowed, all other chars are removed"
 }
+
+variable "prevent_unencrypted_uploads" {
+  type        = "string"
+  default     = "false"
```

```suggestion
  default     = "true"
```

Let's change default to `true` since that's the best practice.

---

done